src/ecs/src/world.rs

```diff
@@ -27,23 +28,43 @@ impl World {
     /// Creates a new entity in the world and returns a handle to it.
     pub fn create_entity(&mut self) -> Entity {
         let id = self.entities.create();
-        self.components.insert(id.clone(), AnyVec::new());
         id
```

Comment:
Since line 30 was removed, the `id` variable is now unnecessary. This can be reduced to simply:

```rust
self.entities.create()
```